src/app/main.cpp

```diff
@@ -0,0 +1,129 @@
+#include "linux_serial_ccsds/linux_serial_ccsds.h"
+
+#include <cstdio>
+#include <unistd.h>
+#include <cstring>
+
+#include "Thread.h"
+
+extern "C"
+{
+#include <Packetizer.h>
+}
+
+static constexpr uint8_t TEXT1[]{ "Hello\n\r" };
+static constexpr uint8_t TEXT2[]{ "Goodbye\n\r" };
+
+static constexpr int SEND_THREAD_PRIORITY1 = 1;
+static constexpr int SEND_THREAD_STACK_SIZE1 = 65536;
+
+static constexpr int SEND_THREAD_PRIORITY2 = 1;
+static constexpr int SEND_THREAD_STACK_SIZE2 = 65536;
+
+static constexpr uint16_t DEVICE1_ID = 0;
+static constexpr uint16_t DEVICE2_ID = 1;
+
+static constexpr size_t NUMBER_OF_DEVICES = 2;
+
+void
+device1_interface_deliver_function(const uint8_t* const data, const size_t data_size)
+{
+    (void)data_size;
+    printf("Serial1 received: %s", data);
+}
+
+void
+device2_interface_deliver_function(const uint8_t* const data, const size_t data_size)
+{
+    (void)data_size;
+    printf("Serial2 received: %s", data);
+}
+
+void* bus_to_driver_private_data[NUMBER_OF_DEVICES];
+void* bus_to_driver_send_function[NUMBER_OF_DEVICES];
+void* interface_to_deliver_function[NUMBER_OF_DEVICES]{ reinterpret_cast<void*>(device1_interface_deliver_function),
+                                                        reinterpret_cast<void*>(device2_interface_deliver_function) };
+void sendThreadMethod1(void* args);
+void sendThreadMethod2(void* args);
+
+Packetizer packetizer{};
```

Comment:
Global variables? Could you create a class, instatiate this class in `main` and run single method?